assets/docs/reboot.css

```diff
@@ -0,0 +1,485 @@
+/*!
```

Comment:
Please add a reference (URL) to where this came from. 

Since this is MIT, we have the choice of either 

- adding the MIT license to the Fuzion repository (how is this usually done when there are several licenses, make LICENSE a directory?)
- or changing the license to GPLv3, which MIT should allow us. 

I prefer to keep this under MIT and add that license to our repository. Can you do this, maybe in a separate patch?

Reply:
@fridis I deleted deleted `reboot.css` now since it was only used when "debugging" the docs and even then it is not super important. Also `flang.dev` has its own `reboot.css` anyway.